goal_src/engine/camera/cam-master.gc

```diff
@@ -6,4 +6,2664 @@
+;      )
+;     (set!
+;      (-> self string-min value z)
+;      (fmin (-> self string-min value z) (+ -4.096 (-> self string-max value z)))
+;      )
+;     (if (nonzero? (-> self no-intro))
+;      (+! (-> self no-intro) -1)
+;      )
+;     (if (nonzero? (-> self force-blend))
+;      (+! (-> self force-blend) -1)
+;      )
+;     (suspend)
+;     )
+;    (none)
+;    )
+;   )
+
+(deftype list-keeper (process)
+  ((dummy float  :offset-assert 112)
+   )
+  :heap-base #x10
+  :method-count-assert 14
+  :size-assert         #x74
+  :flag-assert         #xe00100074
+  )
+
+
+(defstate list-keeper-active (camera-master)
+  :code
+  (behavior ()
+   (while #t
+    (change-to-last-brother self)
+    (suspend)
+    )
+   (none)
+   )
+  )
+
+(defbehavior list-keeper-init camera-master ()
+  (go list-keeper-active)
+  0
+  (none)
+  )
+
+(defbehavior cam-master-init camera-master ()
+  (set! *camera* self)
+  (stack-size-set! (-> self main-thread) 512)
+  (logclear! (-> self mask) (process-mask menu))
+  (set! (-> self master-options) (the-as uint 4))
+  (set! (-> self num-slaves) 0)
+  (dotimes (v1-5 2)
+   (set! (-> self slave v1-5) (the-as (pointer camera-slave) #f))
+   )
+  (set! (-> self slave-options) (the-as uint 560))
+  (set! (-> self view-off-param-save) 1.0)
+  (set! (-> self changer) (the-as uint (process->ppointer self)))
+  (set! (-> self cam-entity) #f)
+  (set! (-> self stringMinHeight) (-> *CAMERA-bank* default-string-min-y))
+  (set! (-> self stringMaxHeight) (-> *CAMERA-bank* default-string-max-y))
+  (set! (-> self stringMinLength) (-> *CAMERA-bank* default-string-min-z))
+  (set! (-> self stringMaxLength) (-> *CAMERA-bank* default-string-max-z))
+  (set! (-> self stringCliffHeight) 163840.0)
+  (set! (-> self no-intro) (the-as uint 0))
+  (set! (-> self force-blend) (the-as uint 0))
+  (set! (-> self string-push-z) (-> *CAMERA-bank* default-string-push-z))
+  (let ((gp-0 (new-stack-vector0)))
+   (set! (-> gp-0 y) (-> *CAMERA-bank* default-string-min-y))
+   (set! (-> gp-0 z) (-> *CAMERA-bank* default-string-min-z))
+   (init! (-> self string-min) gp-0 40.96 409.6 0.9)
+   (set! (-> gp-0 y) (-> *CAMERA-bank* default-string-max-y))
+   (set! (-> gp-0 z) (-> *CAMERA-bank* default-string-max-z))
+   (init! (-> self string-max) gp-0 40.96 409.6 0.9)
+   )
+  (set! (-> self outro-t-step) 0.0)
+  (set! (-> self drawable-target) (the-as handle #f))
+  (set! (-> self which-bone) 0)
+  (let ((a1-3 (new-stack-vector0)))
+   (TODO-RENAME-10 (-> self target-spline) a1-3)
+   )
+  (set! (-> self pov-handle) (the-as handle #f))
+  (set! (-> self pov-bone) 0)
+  (let ((gp-1 (get-process *camera-dead-pool* list-keeper #x4000)))
+   (cond
+    ((when gp-1
+      (let ((t9-5 (method-of-type list-keeper activate)))
+       (t9-5
+        (the-as list-keeper gp-1)
+        self
+        'list-keeper
+        (the-as pointer #x70004000)
+        )
+       )
+      (let ((t9-6 (the-as (function cpu-thread function) set-to-run))
```

Comment:
We shouldn't cast `set-to-run` any more. Hopefully the decompiler will handle this now